### ISSUE TITLE
Provide standalone JavaFX implementations for toolbar actions provided by o.e.g.mvc.fx.ui

### ISSUE THREAD

**Issue by initialflame** [eclipse-gef/gef#22]
If you want to reuse Actions like ZoomInAction, DeleteAction, SelectAllAction from gef.mvc.fx.ui you need to have dependencies to JFace and SWT. 

This is very ugly in case of a want to have a clean JavaFX-Application.

**Comment by mwienand** [comment]
Good catch! The toolbar actions are certainly nice to have in a standalone JavaFX application, too.

The .ui modules may introduce dependencies on Eclipse UI, SWT, and JFace. However, they should only contain integration aspects, such as integration into the Eclipse Undo/Redo History, providing ViewPart and EditorPart implementations that show a GEF Viewer, providing the option to use SWT Controls inside a JavaFX Scene, etc. The main part of the framework does not introduce such dependencies, e.g. o.e.g.mvc.fx, o.e.g.zest.fx, o.e.g.fx, etc.

Rather than remove the integration aspects, we should provide implementations for such toolbar actions for standalone JavaFX applications, too. However, they probably do not belong to o.e.g.mvc.fx. Maybe we need another module. I will rename this issue accordingly.
